cluster/environments/azure/aks-flux/main.tf

```diff
@@ -2,6 +2,29 @@ module "provider" {
     source = "../../../templates/providers/azure/default"
 }
 
+/**
```

Comment:
IMO we should just break out the backend storage into its own module and folder structure as best practice would be to keep this a separate RG or people want to bring their own backend storage account.